primitives/sr-primitives/src/traits.rs

```diff
@@ -713,6 +712,9 @@ pub trait SignedExtension: Codec + Debug + Sync + Send + Clone + Eq + PartialEq
 	/// The type that encodes information that can be passed from pre_dispatch to post-dispatch.
 	type Pre: Default;
 
+	/// An opaque set of information attached to the transaction.
+	type Info: Copy;
```

Comment:
`Copy` is a pretty strict, why not just `Clone`?

Reply:
I did face the dilemma and at the time the rational was: while we are making `Type Info` be generic but we currently know that we'll use it with a type that has `Copy` and this makes the syntax much much easier for now. If we see Copy as merely a marker that clone is cheap, I think it is fair to say that this bundled information will always be cheap to copy.

Reply:
naah made it clone, was easier than what I thought to refactor.